coin-flip/script.js

```diff
@@ -2,12 +2,11 @@ function getRandNumber(max) {
 	return Math.floor(Math.random() * max);
 }
 
-var number = getRandNumber(2);
 
-var headsOrTails = "None";
+const flipCoin = () => {
```

Comment:
Oh, and the `const flipCoin = () => { ... }` syntax is another way to define a function. It's really a shorthand way of what you're doing with `function getRandNumber(max) { ... }`.